test/ensembles.jl

```diff
@@ -189,33 +191,38 @@ MLJBase.info_dict(ensemble_model)
 @test EnsembleModel(atom=ConstantRegressor()) isa Probabilistic
 @test EnsembleModel(atom=MLJModels.DeterministicConstantRegressor()) isa Deterministic
 
-@testset "further test of sample weights" begin
+@testset_accelerated "further test of sample weights" accel begin
     N = 20
     X = (x = rand(3N), );
     y = categorical(rand("abbbc", 3N));
     atom = @load KNNClassifier
     ensemble_model = MLJ.ProbabilisticEnsembleModel(atom=atom,
                                                     bagging_fraction=1,
                                                     n = 5)
-    fitresult, cache, report = MLJ.fit(ensemble_model, 1, X, y)
+    fitresult, cache, report = MLJ.fit(ensemble_model, 1, X, y,
+                                       acceleration=accel)
     @test predict_mode(ensemble_model, fitresult, (x = [0, ],))[1] == 'b'
```

Comment:
Travis is failing because one doesn't control acceleration with a kwarg; rather `acceleration` is a field of `ensemble_model`, so you can set it in the code starting at 200, like this:

```julia
ensemble_model = MLJ.ProbabilisticEnsembleModel(atom=atom,
                                                     bagging_fraction=1,
                                                     n = 5,
                                                     acceleration=accel)
     ```